BUILD.bazel

```diff
@@ -35,6 +35,7 @@ ng_module(
 ng_package(
     # Note: currently this is also the name of the resulting bundles
     name = "ngx-tabs-libdemo",
-    deps = [":root_module"],
+    srcs = [":package.json"],
```

Comment:
ok, `package.json` copies over now. Actually I've seen the entry points are still in the `package.json` from my original, manual setup: https://github.com/juristr/ngx-tabs-libdemo/blob/bazel/package.json#L5-L8

As mentioned in Slack, this would be a point to be discussed IMHO. Whether the `ng_package` rule should also transform the `package.json`, adding the corresponding entry point info in the `package.json` when copying it over to the destination (just like ng-packagr) does.

That'd help reduce errors and make sure the entry points match the produced output files & paths.